robottelo/ui/base.py

```diff
@@ -163,14 +202,9 @@ def delete_entity(self, name, really, name_locator, del_locator,
         # Make sure that element is really removed from UI. It is necessary to
         # verify that fact few times as sometimes 1 second is not enough for
         # element to be actually deleted from DB
+        self.is_entity_delete = True
```

Comment:
Although I am not a fan of this workaround, its okay for now until we find out some other interesting way to do this.